testing/synergy/APKBUILD

```diff
@@ -0,0 +1,73 @@
+# Contributors from Arch Linux's PKGBUILD:
+#  Levente Polyak <anthraxx[at]archlinux[dot]org>                                                                                                                                  
+#  Sven-Hendrik Haase <sh@lutzhaase.com>                                                                                                                                          
+#  Jelle van der Waa <jelle vdwaa nl>                                                                                                                                             
+#  Stéphane Gaudreault <stephane@archlinux.org>                                                                                                                                   
+#  Dale Blount <dale@archlinux.org>                                                                                                                                               
+#  Michael Düll <mail@akurei.me>              
+#
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+pkgname=synergy
+pkgver=1.8.7
+pkgrel=0
+pkgdesc='Share a single mouse and keyboard between multiple computers'
+url='https://symless.com/synergy/'
+arch="all"
+license="GPL2"
+depends="libxtst libxinerama libxkbcommon avahi curl openssl libxrandr"
```

Comment:
abuild will handle the dependencies for shared objects. normally `depends` should be empty.